packages/server/src/services/auth/createUser.ts

```diff
@@ -28,8 +28,11 @@ import error from "../../errorResponse.json";
  * @returns {object|null} - Returning user data object from database or null
  */
 const createUser = async (req, res, _next, userData) => {
+  // change email to lowercase to avoid case-sensitivity
+  const lowerCaseEmail = userData.email.toLowerCase();
+
   // generate user unique identification
-  const userId = uuidv4(userData.email);
+  const userId = uuidv4(lowerCaseEmail);
```

Comment:
I don't recall myself, why I passed the email as argument to uuid generator function.

_must be long time ago_ 😅